ChangeLog

```diff
@@ -232,6 +260,9 @@
 - Fix unhealthy usage of `split` in language menu (#5241)
 - Skip installing locales when translations are disabled (#5302)
 - Tut01 String Fix (#5327)
+- Fix string bugs in lore texts reported on Transifex (#5399)
+- Fix Transifex String Issues (#5462)
```

Comment:
I'm not sure if these belong in I18n, they're just typo and punctuation fixes.

Reply:
That's where we always put these entries, but I grouped it with another one now